tensorflow_datasets/core/dataset_builder.py

```diff
@@ -61,16 +79,23 @@
 """
 
 
-class BuilderConfig(object):
+class BuilderConfig(object):  # pylint: disable=useless-object-inheritance
   """Base class for `DatasetBuilder` data configuration.
 
   DatasetBuilder subclasses with data configuration options should subclass
   `BuilderConfig` and add their own properties.
   """
+  _name: str
+  _version: str
+  _supported_versions: List[str]
+  _description: str
```

Comment:
From https://www.python.org/dev/peps/pep-0526/#class-and-instance-variable-annotations:

> As a matter of convenience (and convention), instance variables can be annotated in __init__ or other methods, rather than in the class:
>
> ```python
> class Box(Generic[T]):
>    def __init__(self, content):
>        self.content: T = content
> ```

It should work in the `__init__`. if it doesn't there is either an issue in Pytype or in the annotations or in our code. 

Could you try `__init__` on a standalone example ? Then here without the `@api_utils.disallow_positional_args` decorator ?
We want annotations in `__init__` so we should try to understand why this error is raised.

Reply:
I have removed unnecessary code. Can you please run this and check it plz.